### PR TITLE
Feature/downtime support

### PR DESCRIPTION
**Pull request by jnation3406**

### CODE REVIEW

File: valhalla/settings.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
 """
 
 import os
+import sys
```
Comment (Fingel) [inline valhalla/settings.py]: This doesn't look used.

File: valhalla/common/downtimedb.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                data = self._get_downtime_data()
+                self.downtime_intervals = self._order_downtime_by_resource(data)
+                caches['locmem'].set('downtime_intervals', self.downtime_intervals, 900)
+            except DowntimeDBException as e:
+                logger.warning(repr(e))
+
+        return self.downtime_intervals
+
+
+downtimedb = DowntimeDB()
```
Comment (Fingel) [inline valhalla/common/downtimedb.py]: Are you sure we want this to be a global? What's wrong with instantiating the class when we need it?

File: valhalla/settings.py [hunk 3]
```diff
@@ -110,7 +111,7 @@
         'LOCATION': os.getenv('CACHE_LOCATION', 'unique-snowflake')
     },
     'locmem': {
-        'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
+        'BACKEND': os.getenv('LOCAL_CACHE_BACKEND', 'django.core.cache.backends.dummy.DummyCache'),
```
Comment (Fingel) [inline valhalla/settings.py]: Need to remember to set this now when it's deployed to prod.

File: valhalla/common/downtimedb.py (hunk 2 above)
Comment (jnation3406) [reply valhalla/common/downtimedb.py]: removed global and changed to static class usage

File: valhalla/settings.py (hunk 1 above)
Comment (jnation3406) [reply valhalla/settings.py]: removed

**Pull request merged**